.github/pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+# Description:
+
+Briefly describe the change of behavior
+
+
+- [ ] Closes #xxxx
+- [ ] Tests added / passed
+- [ ] Passes `./format_code.sh`
```

Comment:
Why do we add the two above? They are covered by CI?

Reply:
I took it as a template from pandas. I thought that was a nice reminder but I can remove it again, of course